spek-ide-plugin/interop-jvm/src/main/kotlin/org/spekframework/ide/sm.kt

```diff
@@ -0,0 +1,112 @@
+package org.spekframework.ide
+
+import org.spekframework.spek2.runtime.execution.ExecutionListener
+import org.spekframework.spek2.runtime.execution.ExecutionResult
+import org.spekframework.spek2.runtime.scope.ActionScopeImpl
+import org.spekframework.spek2.runtime.scope.GroupScopeImpl
+import org.spekframework.spek2.runtime.scope.TestScopeImpl
+import java.io.CharArrayWriter
+import java.io.PrintWriter
+
+class ServiceMessageAdapter: ExecutionListener {

```

Comment:
Rename file from `sm.kt` to `ServiceMessageAdapter`?

Reply:
👍